src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -63,12 +63,14 @@
 </p>
 
 <p>
-  For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}.
-</p>
-
-${additionalInstructions}
-<p>
-  If you need any help or clarifications regarding TEAMMATES, please email us at ${supportEmail}.
+  <ul>
+    ${additionalInstructions}
+    <li><strong>Requests for deadline extensions, problems regarding team/student data</strong> (e.g. wrong team, misspelled name),
+    and <strong>other non-technical queries</strong> about the feedback session :
+    <br>Contact the instructors of the course: ${coOwnersEmails}</li>
+    <li><strong>Technical help regarding TEAMMATES</strong> (e.g. submission link is not working) :
+    Email TEAMMATES support team at ${supportEmail}.</li>
+  </ul>
```

Comment:
This entire block is duplicated in 5 other files. See if you can make a template out of it.

Reply:
Okay, I will try to change this and make it a template.